### PR TITLE
Attention rollout

### PR DESCRIPTION
**Pull request by Gasp34**

### CODE REVIEW

File: collections/_posts/2022-07-07-AttentionRollout.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+In Attention Flow, the weights are considered as **the capacity** of the edge. In graph theory, the flow of the graph must satisfies two conditions : 
+* **Capacity constraint** : for each edge the flow should not exceed its capacity
+* **Flow conservation** : for all nodes (except source and target) the input flow is equal to the output flow
+
+Using any maximum flow algorithm it is possible to compute the maximum attention flow from any nodes of a hidden layers to any input node.
+
+# Experiment
+
+* Task : verb number prediction (i.e. singularity of plurality of a verb in a sentence)
```
Comment (nathanpainchaud) [inline collections/_posts/2022-07-07-AttentionRollout.md:70]: Typo? I think you meant to write _or_ and not _of_?
```suggestion
* Task : verb number prediction (i.e. singularity or plurality of a verb in a sentence)
```

**Review by nathanpainchaud — CHANGES_REQUESTED** [review]
Good review! Just a few typos/mistakes :wink:

File: collections/_posts/2022-07-07-AttentionRollout.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+To compute the attention from layer $$i$$ to layer $$j$$ we can use this equation : 
+<center>
+$$
+\tilde{A}\left(l_{i}\right)=\left\{\begin{array}{ll}
+A\left(l_{i}\right) \tilde{A}\left(l_{i-1}\right) & \text { if } i>j \\
+A\left(l_{i}\right) & \text { if } i=j
+\end{array}\right.
+$$
+</center>
```
[the remark this answers is not in the document]
Comment (Gasp34) [reply collections/_posts/2022-07-07-AttentionRollout.md:57]: for some reason it was not centered without the html tag

[the remark this answers is not in the document]
Comment (Gasp34) [reply collections/_posts/2022-07-07-AttentionRollout.md]: That's right, thanks!

**Comment by Gasp34** [comment]
Thx Nathan ! I applied all your suggestions

**Review by nathanpainchaud — APPROVED** [review]
Thanks for the rapid responses ;)

**Pull request merged**